src/app/component/layout/components/header/header.component.scss

```diff
@@ -42,7 +40,6 @@ $screen-320: " screen and (max-width : 320px)";
       width: 136px;
       height: 36px;
       margin-right: 4%;
```

Comment:
I know that isn't your code but we shouldn't mix up the  `%` and `px`.

Reply:
> I know that isn't your code but we shouldn't mix up the `%` and `px`.

Fixed: changed to pixels